flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGateTest.java

```diff
@@ -812,242 +788,13 @@ public void testQueuedBuffers() throws Exception {
 		}
 	}
 
-	@Test
-	public void testBufferReceivedListener() throws Exception {
```

Comment:
Can you double check if indeed all of those test should be removed? I don't see how things like `testPartitionNotFoundExceptionWhileGetNextBuffer` should be related to this commit/PR?

Reply:
Good catch, my intent was to delete `testBufferReceivedListener` and `testPartitionNotFoundExceptionWhileGetNextBuffer` but not the test in between them.

`testBufferReceivedListener` tests `BufferReceivedListener` which this commits renders useless (and is later removed).

`testPartitionNotFoundExceptionWhileGetNextBuffer` tests concurrent spilling of lingering buffers and receiving of such lingering buffers. Both now happens in the same thread, so the test does not make any sense.